module/x/peggy/keeper/querier_test.go

```diff
@@ -871,3 +871,14 @@ func TestQueryDenomToERC20(t *testing.T) {
 
 	assert.Equal(t, erc20, string(queriedERC20))
 }
+
+func TestQueryInjDenomToERC20(t *testing.T) {
+	input := CreateTestEnv(t)
+	ctx := input.Context
+	injContractAddr := input.PeggyKeeper.GetInjContractAddress(ctx)
```

Comment:
Token denom should be a param.

`QueryInjDenomToERC20` -> `QueryCosmosCoinERC20` which returns both denom and erc20 contract address. Make test name matching the keeper's function name.